src/Security.tsx

```diff
@@ -61,10 +56,15 @@ const Security: React.FC<{
 
     // Trigger an initial change event to make sure authState is latest
     if (!oktaAuth.isLoginRedirect()) {
-      oktaAuth.authStateManager.updateAuthState();
+      // Calculates initial auth state and fires change event for listeners
```

Comment:
Why are we only starting service if not callback? When returning on the callback, the SPA app may go to different routes, when does the service get started in this case? I'm thinking it should start in all cases. What do you think @shuowu ?

Reply:
Added service start after `handleLoginRedirect`